<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OmrPanel</class>
 <widget class="QWidget" name="OmrPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="spatium">
     <property name="suffix">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Distance between two lines on a standard 5-line staff</string>
     </property>
     <property name="whatsThis">
      <string>Distance between two lines on a standard 5-line staff</string>
     </property>
     <property name="text">
      <string>Staff space:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="process">
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="showStaves">
     <property name="text">
      <string>Mark staves</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="showSlices">
     <property name="text">
      <string>Mark slices</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="showBarlines">
     <property name="text">
      <string>Mark barlines</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="showLines">
     <property name="text">
      <string>Show lines</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>showLines</tabstop>
  <tabstop>showBarlines</tabstop>
  <tabstop>showSlices</tabstop>
  <tabstop>showStaves</tabstop>
  <tabstop>spatium</tabstop>
  <tabstop>process</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
